UnrealPlugin/ThirdPersonMPBuild.md

```diff
@@ -0,0 +1,70 @@
+
+### Development Editor
+
+The development editor should be the first thing you build. To do this, find your ThirdPersonMP.uproject file, right click, and "Generate Visual Studio project files" (You may have already done this in the previous guide, in which case, you can skip this step. Once complete, you can open ThirdPersonMP.sln in Visual Studio.
+
+Once Visual Studio loads, change your configuration to "Development Server". The Solution Configuration dropdown is sometimes hard to read, but can be resized in the toolbar customization options. Now, build. The first time you do this, it can take several hours, but you likely did this in the previous guide. Repeated builds should be fairly fast for a small project. Once finished, you can run/debug the project Solution/Games/ThirdPersonMP. This will start the Development Editor.
+
+### Shipping Client
+
+Run the Development Editor, and from the development editor window, pick these options in this order:
+
+* File -> Package Project -> Build Target -> ThirdPersonMP (Or the name of your project)
+* File -> Package Project -> Build Configuration -> Development
+* File -> Package Project -> Windows (64-bit)
+* (This opens a popup window asking you to pick a location - make sure you remember this location)
+* {Wait a very long time}
+	* The first shipping client build will take upto multiple hours, even if you've already built Development Editor
+	* Repeat builds will be much faster for a simple project like ThirdPersonMP
+* Open the client location you picked and find ThirdPersonMP.exe
+
+### Development Server
+
+* Close the Development Editor (Unreal is no longer running, only Visual Studio should be open now)
+* Toolbars -> Configuration -> "Development Server"
+* Build/Rebuild Solution
+* {Wait a very long time}
+	* The first development server build will take upto multiple hours, even if you've already built Development Editor
+	* Repeat builds will be much faster for a simple project like ThirdPersonMP
+* Use Explorer to find {projectLocation}\Binaries\Win64\ThirdPersonMPServer.exe
+
+### Shipping Server
+
+Run the Development Editor, and from the development editor window, pick these options in this order:
+
+* File -> Package Project -> Build Target -> ThirdPersonMPServer (Or the name of your project)+Server
+* File -> Package Project -> Build Configuration -> Development
+* File -> Package Project -> Windows (64-bit)
+* (This opens a popup window asking you to pick a location - make sure you remember this location)
+* {Wait a very long time}
+	* The first shipping server build will take upto multiple hours, even if you've already built Development Editor
+	* Repeat builds will be much faster for a simple project like ThirdPersonMP
+* Open the server location you picked and find ThirdPersonMPServer.exe
+
+## Navigation
+
+You are now ready to [deploy your game server](ThirdPersonMPLocalDeploy.md) to PlayFab MPS and test your project in the cloud.
```

Comment:
as in the previous comment, I'd recommend guiding people to use LocalMultiplayerAgent

Reply:
Good catch. The file link tells them to do exactly that, but the description here is misleading. Thanks.